include/emp/web/Div.hpp

```diff
@@ -368,6 +369,8 @@ namespace web {
     }
 
     // A quick way to retrieve Animate widgets by name.
+    // TODO: may want to change this to use .at() rather than [] since this default
```

Comment:
We'd have to handle exceptions with `.at()` since if no value exists for the key it throws one. Problem with `[]` is it will default construct an Animator if there isn't one there already which probably isn't ideal behavior.